packages/stylable/src/stylable.ts

```diff
@@ -66,6 +67,7 @@ export class Stylable {
         protected scopeRoot: boolean = true,
         protected resolveOptions: any = {},
         protected optimizer?: StylableOptimizer,
+        protected mode: 'production' | 'development' = 'production',
```

Comment:
doesn't the plugin defaults to development? is this set to production in order to avoid the warning rule in tests?

Reply:
I don't think that having visual warnings pop up for a user without him asking for `dev` mode explicitly would be a good idea, hence defaulting to `production`.

Reply:
I agree, but the default for the webpack plugin is `dev`, so if we agree that `production` is a better default, then lets change the webpack-plugin as well (just ask @barak007).

Reply:
I don't believe the `webpack-plugin` itself has a default. We have a default in `stylable-scripts` for the `start`/`build` scripts which get resolved to `development`/`production` modes respectively.

If someone were to just bring `stylable` and `stylable-webpack-plugin` as dependencies, whatever they'll write in their webpack config will be the default.